docs/conf.py

```diff
@@ -32,13 +43,23 @@
 sys.path.insert(0,os.path.abspath('../openpathsampling/'))
 #sys.path.append(os.path.abspath('_themes'))
 
+# -- Preparing the CLI docs -----------------------------------------------
+orig_cwd = os.getcwd()
+try:
+    cli_input_dir = os.path.abspath("./cli/compile/input/")
```

Comment:
shouldn't this directory be relative to the current file instead of the cwd?

Reply:
Done in [b9e2818](https://github.com/openpathsampling/openpathsampling/pull/1159/commits/b9e281856d3dcd9758bffcf902fcecc44480f0d7)